templates/home/index.html.twig

```diff
@@ -104,6 +104,29 @@
         </div>
     </section>
 
+    <section class="container-fluid badges text-center">
+        <div class="container">
+            <div class="row d-flex">
+                {% for badge in badges.badge_styles %}
+                    <div class="col-12 col-md-6">
+                        <h4>{{ badge.label }}</h4>
+                        <a href="{{ badge.imgLink }}" target="_blank">
+                            <img class="badge" src="{{ badge.img }}">
+                        </a>
+
+                        <input
+                                class="badge-input"
+                                data-badge="{{ badge.name }}"
+                                readonly
+                                type="text"
+                                value="{{ badge.markdown }}" title="{{ badge.label }}">
+                        <button data-clipboard-target=".badge-input[data-badge='{{ badge.name }}']">Copy</button>
+                    </div>
+                {% endfor %}
+            </div>
+        </div>
+    </section>
```

Comment:
can you put a screenshot please! :)

Reply:
Will do once I get the correct badges rendered :)

Reply:
WDYT? :)

![Schermata 2022-02-18 alle 18 12 07](https://user-images.githubusercontent.com/190820/154730234-855d8a13-9813-4b2d-a9d9-8d1f38171b83.png)

Reply:
First the flat variants, then plastic and for-the-badge. Speaking of, can you update PUGX/poser to also include the fourth style, please? :D

Reply:
I tried to do composer update package but I've a error! :S

Reply:
```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires badges/poser ^2.3 -> satisfiable by badges/poser[v2.3.0].
    - badges/poser v2.3.0 requires kartsims/easysvg ^2.0 -> found kartsims/easysvg[dev-master, 1.0] but it does not match the constraint.
```

Reply:
We need to add my fork as a repository as well, it seems...

```
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/ricardoboss/easysvg"
        }
    ],
```